assets/src/edit-story/components/checklistCard/stories/index.js

```diff
@@ -0,0 +1,350 @@
+                type={THUMBNAIL_TYPES.IMAGE}
+                displayBackground={THUMBNAIL_BG[THUMBNAIL_TYPES.IMAGE]}
+                aria-label="my helper text describing this thumbnail image"
+              />
+            </>
+          }
+        />
+      </div>
+      <div>
+        <Text>{'Multiple Issues'}</Text>
+        <ChecklistCard
+          title="Add video captions"
+          cta={
+            <Button size={BUTTON_SIZES.SMALL} type={BUTTON_TYPES.SECONDARY}>
+              {'Fix everything'}
+            </Button>
+          }
+          footer={
+            <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
+              {
+                'Keep the audience engaged even when they can’t listen to the audio. '
+              }
+              <Link
+                href="/demo"
+                size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+              >
+                {'Learn more'}
+              </Link>
+            </Text>
+          }
+          cardType={CARD_TYPE.MULTIPLE_ISSUE}
+          thumbnailCount={3}
+          thumbnail={
+            <>
+              <Thumbnail
+                onClick={() => action('thumbnail action found')()}
+                type={THUMBNAIL_TYPES.IMAGE}
+                displayBackground={THUMBNAIL_BG[THUMBNAIL_TYPES.IMAGE]}
+                aria-label="my helper text describing this thumbnail image"
+              />
+              <Thumbnail
+                onClick={() => action('thumbnail action found')()}
+                type={THUMBNAIL_TYPES.IMAGE}
+                displayBackground={THUMBNAIL_BG[THUMBNAIL_TYPES.IMAGE]}
+                aria-label="my helper text describing this thumbnail image"
+              />
+              <Thumbnail
+                onClick={() => action('thumbnail action found')()}
+                type={THUMBNAIL_TYPES.IMAGE}
+                displayBackground={THUMBNAIL_BG[THUMBNAIL_TYPES.IMAGE]}
+                aria-label="my helper text describing this thumbnail image"
+              />
+            </>
+          }
+        />
+      </div>
+
+      <div>
+        <Text>{'Multiple Issues - more than 4'}</Text>
+        <ChecklistCard
+          title="Add video captions"
+          cta={
+            <Button size={BUTTON_SIZES.SMALL} type={BUTTON_TYPES.SECONDARY}>
+              {'Fix everything'}
+            </Button>
+          }
+          footer={
+            <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
+              {
+                'Keep the audience engaged even when they can’t listen to the audio. '
+              }
+              <Link
+                href="/demo"
+                size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+              >
+                {'Learn more'}
+              </Link>
+            </Text>
+          }
+          cardType={CARD_TYPE.MULTIPLE_ISSUE}
+          thumbnailCount={6}
+          thumbnail={
```

Comment:
Depending on how we want to handle rendering thumbnails the grid might need more work. 

if we want to only ever render 3 (if there's more than 4) thumbnails then we're fine, but if we're rendering alllll of the thumbnails then I'll need to tend to the card component's grid a bit more. Right now the overflow's just hidden but the thumbnails will double up if you render more than the row allows. One of those things that feels easier to look at in isolation and didn't think it would matter to the other updates going on in PPC too much at the moment.